index.php

```diff
@@ -9,10 +9,64 @@
     <div class="wrapper">
         <div id="logo-bar">
             <?php include 'logo-include.php'; ?>
-            <h1> Flock Information Database Home </h1>
-        </div>        
+            <h1> Home </h1>
+        </div>
+
         <div class="content">
-            <h2> Stick a login form here </h2>
+            <table id="register-login">
+            <tr>
+            <td>
+                <form id="register-form" method="post" action="index.php">
+                    <table id="inner-register">
+                        <th colspan="2"><h3>New users register here.</h3></th>
+                        <tr>
+                            <td><label for="register_username"><b>Username<b></label></td>
+                            <td><input type="text" name="username"></td>
+                        </tr>
+                        <tr>
+                            <td><label for="register_password"><b>Password</label></b></td>
+                            <td><input name="password" type="password"></input></td>
+                        </tr>
+                        <tr>
+                            <td><label for="user_type"><b>User Type</b></label></td>
+                            <td><input type="text" name="user_type"></td>
+                        </tr>
+                        <tr>
+                            <td><label for="email"><b>Email</b></label></td>
+                            <td><input type="text" name="email"></td>
+                        </tr>
+                        <tr>
+                            <td colspan="2"><input type="submit" value="Register"/></td>
+                        </tr>
+                    </table>
+                </form>
+            </td>
+
+            <td>
+                <form id="login-form" method="post" action="redirectPostLogin.php" >
+                    <table id="inner-login">
+                        <th colspan="2"><h3>Existing users login here.</h3></th>
+                        <tr>
+                            <td><label for="login_username"><b>Username</b></label></td>
+                            <td><input type="text" name="username"></td>
```

Comment:
The `for` in `label` needs to match the name of the input you're labelling. That said, I actually like `login_username` better than `username` due to the conflict with the same `username` above.  So I think change the input name to `login_password` etc.